tests/ui/lint/dropping_copy_types-issue-125189.rs

```diff
@@ -0,0 +1,17 @@
+//@ check-fail
```

Comment:
It would be great if we could test the applicability of the suggestion with `rustfix`. This can be achieved with `run-rustfix` and having no others warnings/errors (ie move the match in a different UI test file).

```suggestion
//@ check-fail
//@ run-rustfix
```

Reply:
Done. Thank you.